content/docs/user-guide/assets/asset-types.md

```diff
@@ -6,26 +6,47 @@ weight: 700
-| `.stl` | A simple mesh format used in computer aided design (CAD) and 3D printing. Unlike other 3D scene formats, `.stl` assets can only contain a simple model with limited vertex attributes. | `.azmodel`</br>`.azlod`</br>`.azbuffer`</br>`.pxmesh` |
\ No newline at end of file
+| Source Asset | Builder | Builder Location | Description  | Product Assets | 
+| - | - | - | - | - |
+| `.animgraph` | AnimGraphBuilderWorker | [EMotionFX Gem](https://github.com/o3de/o3de/tree/development/Gems/EMotionFX) | [Animation graphs](docs/user-guide/visualization/animation/character-editor/concepts-and-terms) define animation behavior for characters. | `.animgraph` |
+| `.attimage`, `.azasset`, `.azbuffer` | Any Asset Builder | [Atom Gem](https://github.com/o3de/o3de/blob/development/Gems/Atom/RPI/Code/Source/RPI.Builders/Common/AnyAssetBuilder.h) | - | Same as input, `.attimage` generates an `.attimage` product asset. |
+| `.texatlas` | Atlas Worker Builder | [Texture Atlas Gem](https://github.com/o3de/o3de/blob/development/Gems/TextureAtlas/Code/Source/Editor/AtlasBuilderWorker.h) | [Texture atlases](docs/user-guide/interactivity/user-interface/canvases/texture-atlases) are used to reduce draw calls in user interfaces. | `.texatlasidx` and `.streamingimage` |
+| `.bmp`, `.dds`, `.exr`, `.gif`, `.jpeg`, `.jpg`, `.png`, `.tga`, `.tiff`, `.tif` | Atom Image Builder | [Atom Gem](https://github.com/o3de/o3de/blob/development/Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageBuilderComponent.h) | Processes images for use in O3DE, primarily for use as [Textures](docs/user-guide/assets/texture-settings/texture-assets). | Output depends on settings, but will generally include `.imagemipchain` and `.streamingimage` |
+| `.materialtype`, `.material` | Atom Material Builder | [Atom Gem](https://github.com/o3de/o3de/blob/development/Gems/Atom/RPI/Code/Source/RPI.Builders/Material/MaterialBuilder.h) | [Materials](docs/atom-guide/look-dev/materials) contain properties to define how objects are rendered. | `.materialtype` outputs `.material` and `.azmaterialtype` products, `.material` outputs `.azmaterial` products. |
+| `.resourcepool` | Atom Resource Pool Asset Builder | [Atom Gem](https://github.com/o3de/o3de/blob/development/Gems/Atom/RPI/Code/Source/RPI.Builders/ResourcePool/ResourcePoolBuilder.h) | - | `.pool`, or `.streamingimagepool` |
+| `.benchmarksettings`, `.benchmark` | Benchmark Asset Worker Builder | [LmbrCentral Gem](https://github.com/o3de/o3de/blob/development/Gems/LmbrCentral/Code/Source/Builders/BenchmarkAssetBuilder/BenchmarkAssetBuilderComponent.h) | - | - |
+| `.blast` | Blast Scene Builder | [Blast Gem](https://github.com/o3de/o3de/blob/development/Gems/Blast/Editor/Scripts/blast_asset_builder.py) | - | `.blast`, `.assetinfo.generated` |
+| `.cfg` | CFG Builder Worker | [LmbrCentral Gem](https://github.com/o3de/o3de/blob/development/Gems/LmbrCentral/Code/Source/Builders/CopyDependencyBuilder/CfgBuilderWorker/CfgBuilderWorker.h) | - | `.cfg` |
```

Comment:
is this still a thing?

Reply:
The builder is still there and registered. I gathered this list of assets from what shows up for Automated Testing right now.